metrics/storage/webtooling.sh

```diff
@@ -0,0 +1,194 @@
+	CONTAINERS_ID=($(sudo ctr c list -q))
+	for i in "${CONTAINERS_ID[@]}"; do
+		sudo ctr t exec --exec-id "$(random_name)" "$i" sh -c "$RESULTS_CMD" >> "$TMP_DIR"/results
+	done
+
+	# Save configuration
+	metrics_json_start_array
+
+ 	local output=$(cat "$TMP_DIR/results")
+ 	local cut_results="cut -d':' -f2 | sed -e 's/^[ \t]*//'| cut -d ' ' -f1 | tr '\n' ',' | sed 's/.$//'"
+
+	local acorn=$(echo "$output" | grep -w "acorn" | eval "${cut_results}")
+	local babel=$(echo "$output" | grep -w "babel" | sed '/babel-minify/d' | eval "${cut_results}")
+	local babel_minify=$(echo "$output" | grep -w "babel-minify" | eval "${cut_results}")
+ 	local babylon=$(echo "$output" | grep -w "babylon" | eval "${cut_results}")
+	local buble=$(echo "$output" | grep -w "buble" | eval "${cut_results}")
+ 	local chai=$(echo "$output" | grep -w "chai" | eval "${cut_results}")
+ 	local coffeescript=$(echo "$output" | grep -w "coffeescript" | eval "${cut_results}")
+ 	local espree=$(echo "$output" | grep -w "espree" | eval "${cut_results}")
+ 	local esprima=$(echo "$output" | grep -w "esprima" | eval "${cut_results}")
+ 	local jshint=$(echo "$output" | grep -w "jshint" | eval "${cut_results}")
+ 	local lebab=$(echo "$output" | grep -w "lebab" | eval "${cut_results}")
+	local postcss=$(echo "$output" | grep -w "postcss" | eval "${cut_results}")
+ 	local prepack=$(echo "$output" | grep -w "prepack" | eval "${cut_results}")
+	local prettier=$(echo "$output" | grep -w "prettier" | eval "${cut_results}")
+	local source_map=$(echo "$output" | grep -w "source-map" | eval "${cut_results}")
+ 	local terser=$(echo "$output" | grep -w "terser" | eval "${cut_results}")
+ 	local typescript=$(echo "$output" | grep -w "typescript" | eval "${cut_results}")
+ 	local uglify_js=$(echo "$output" | grep -w "uglify-js" | eval "${cut_results}")
+ 	local geometric_mean=$(echo "$output" | grep -w "Geometric" | eval "${cut_results}")
+ 	local tps=$(echo "$geometric_mean" | sed "s/,/+/g;s/.*/(&)\/$NUM_CONTAINERS/g" | bc -l)
+  	local total_tps=$(echo "$tps*$NUM_CONTAINERS" | bc -l)
+
+  	local json="$(cat << EOF
+  	{
+ 		"Acorn" : "$acorn",
+  		"Babel" : "$babel",
+   		"Babel minify" : "$babel_minify",
+   		"Babylon" : "$babylon",
+    		"Buble" : "$buble",
+  		"Chai" : "$chai",
+   		"Coffeescript" : "$coffeescript",
+   		"Espree" : "$espree",
+		"Esprima" : "$esprima",
+ 		"Jshint" : "$jshint",
+  		"Lebab" : "$lebab",
+  		"Postcss" : "$postcss",
+  		"Prepack" : "$prepack",
+    		"Prettier" : "$prettier",
+    		"Source map" : "$source_map",
+     		"Terser" : "$terser",
+    		"Typescript" : "$typescript",
+    		"Uglify js" : "$uglify_js",
+     		"Geometric mean" : "$geometric_mean",
+     		"TPS" : "$tps",
+      		"Total TPS" : "$total_tps"
+       	}
+EOF
+)"
+	metrics_json_add_array_element "$json"
+	metrics_json_end_array "Results"
+	metrics_json_save
+	sudo ctr tasks rm -f $(sudo ctr task list -q)
+	sudo ctr c rm $(sudo ctr c list -q)
+	sudo rm -rf "${tmp_data_dir}"
```

Comment:
`tmp_data_dir` isn't defined fwics? Maybe this should be `$TMP_DIR`? But I think you can just remove this line as the script is always going to call `remove_tmp_dir()` on exit.